build.gradle

```diff
@@ -432,6 +432,7 @@ subprojects {
                  'opencensus-contrib-log-correlation-log4j2',
                  'opencensus-contrib-log-correlation-stackdriver',
                  'opencensus-contrib-monitored-resource-util',
```

Comment:
Add a TODO to remove this from `uploadArchives`?

Reply:
Create https://github.com/census-instrumentation/opencensus-java/issues/1748 to track everything.